extensions/amp-date-picker/validator-amp-date-picker.protoascii

```diff
@@ -200,6 +202,11 @@ attr_lists: {
   }
 }
 
+attr_lists: {
+  name: "amp-date-picker-overlay-mode-attributes"
+  attrs: { name: "touch-keyboard-editable" }
```

Comment:
It doesn't look like this will have any values so we should probably add `value: ""`.

Reply:
Good call, will do